custom/icds_reports/models/aggregate.py

```diff
@@ -347,6 +348,8 @@ class ChildHealthMonthly(models.Model, AggregateMixin):
     lunch_count = models.IntegerField(blank=True, null=True)
     state_id = models.TextField(blank=True, null=True)
     opened_on = models.DateField(blank=True, null=True)
+    birth_weight = models.PositiveSmallIntegerField(null=True, help_text="birth weight in grams")
+    child_id = models.TextField(blank=True, null=True)
```

Comment:
What is child_id?

Reply:
Is it the person case_id of child?
May be something like `child_person_case_id` would be clearer?

Reply:
yeah